.gitignore

```diff
@@ -34,3 +34,4 @@
 /ci/
 Puppetfile.lock
 *.pyc
+.vscode/
```

Comment:
FYI if you want this change to stick, need to add it to `.sync.yml` here: https://github.com/StackStorm/puppet-st2/blob/master/.sync.yml#L2